api_spec.yaml

```diff
@@ -115,7 +115,7 @@ paths:
                                 email:
                                   type: string
                                   example: "johndoe@gmail.com"
-                                phone:
+                                phoneNumber:
```

Comment:
This changes to reflect the correct property naming based on the returned data from API